docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,178 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(masternode_announce_message)
```

Comment:
The parameter names are important and form part of the RPC API, and for that reason the code and the documentation must match for parameter names.  This parameter name is too long and in the code it is "signmb".

How about just "message"?

I'm curious - why is Electrum Dash responsible for broadcasting masternode announcements?  Isn't this something the Dash daemon does itself?

Reply:
Ok I'm going to update the parameter names in order to match the docs and the code.

No, it doesn't, every time you create a masternode you have to create an announcement message and broadcast it to the network manually.

And thanks for the RPC explanation I didn't know that.